Program.cs

```diff
@@ -1,10 +1,20 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Find LCM and GCD of two numbers
+
+Console.Write ("Enter the first number: ");
+string first_number = Console.ReadLine ();
+Console.Write ("Enter the second number: ");
+string second_number = Console.ReadLine ();
+int number1 = int.Parse (first_number);
+int number2 = int.Parse (second_number);
+int[] multiples = new int[] { };

```

Comment:
change this to list. Here you are creating a new array for each and every append which is not correct.